### ISSUE TITLE
Knowledgebase in gridview is not mapping to primary name

### ISSUE THREAD

**Issue by lonnylundsten** [mbari-org/vars-gridview#31]
Gridview should be able to get the primary name for all concepts. In cases when a common name or secondary name is typed in, the primary name should be used for the class label.

**Comment by hohonuuli** [comment]
Example kb call to get alternate names: <http://m3.shore.mbari.org/kb/v1/concept/crab>

```json
{
  "name": "Decapoda",
  "alternateNames": [
    "prawn",
    "crayfish",
    "lobster",
    "shrimp",
    "crab"
  ],
  "rank": "order",
  "media": [],
  "descriptors": [
    {
      "linkName": "is-bioluminescent",
      "toConcept": "self",
      "linkValue": "uncertain-unlikely"
    }
  ],
  "author": "unknown"
}
```

**Comment by lonnylundsten** [comment]
VL wants GridView to accept any alias name and automatically enter primary concept name and carry forward into VARS.
Hoping to have completed by 11/30/2023.

**Comment by kevinsbarnard** [comment]
Can do -- I'll prioritize this task while the VL is testing the current S2 integration.

**Comment by kevinsbarnard** [comment]
Addressed in v0.3.17

**Comment by kevinsbarnard** [comment]
@lonnylundsten, this remapping is now performed automatically when renaming an observation. VARS Gridview will hit the endpoint that @hohonuuli provided above to get the `name` and apply that when updating the concept. So for example, typing "crab" in the concept dropdown and hitting the "label" button will result in the selected bounding boxes' observations to be renamed to "Decapoda".
